.github/workflows/ci.yml

```diff
@@ -10,6 +10,46 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  build-others:
```

Comment:
NOTE: not running plugin test/ tests because i cannot make it run on windows. Blame it.
I am tired of playing with the CI to test windows alone.